### DISCUSSION TITLE
Setting server Timezone

### DISCUSSION THREAD

**Discussion by russ-sentient** [wolveix/satisfactory-server#347] [Q&A] [answered]
I am having trouble setting the containers timezone.  I have tried adding the ENV variable TZ with value America/Chicago and have tried mapping the host's /etc/locatime  path to the containers /etc/localtime.  Neither seem to work, it is still showing UTC time when I pull up a docker cli and run the 'date' cmd.  I wouldn't care, but the server restart happens at 6pm local time every day, right in the middle of when I am playing after work and I was hoping to get that to happen at midnight localtime rather than midnight UTC.

Thank you for the awesome container, it has been running great.

**Comment by russ-sentient** [comment]
I did comb through the Dockerfile and noticed that the tzdata package was added to the image build, not sure what I am missing.

![image](https://github.com/user-attachments/assets/5ebba646-d94a-4308-80e0-4dcf313bae74)

![image](https://github.com/user-attachments/assets/c0aa274e-fd08-4eb9-92ff-e271efabf900)

**Reply by wolveix** [reply, marked as answer]
@russ-sentient the reason is that the timezone patch hasn't been pushed to any of the Docker images yet. There's an [outstanding issue with Docker Hub](https://www.dockerstatus.com/pages/incident/533c6539221ae15e3f000031/66f719d334b03d6913767794), preventing me from pushing the fix there. In the meantime, I've just manually build and pushed `ghcr.io/wolveix/satisfactory-server:latest` which uses GitHub's package registry. If you pull that, it should work for you.

**Reply by russ-sentient** [reply]
Well, that would explain it!  Thank you!

**Reply by russ-sentient** [reply]
> @russ-sentient the reason is that the timezone patch hasn't been pushed to any of the Docker images yet. There's an outstanding issue with Docker Hub, preventing me from pushing the fix there. In the meantime, I've just manually build and pushed `ghcr.io/wolveix/satisfactory-server:latest` which uses GitHub's package registry. If you pull that, it should work for you.

That worked perfectly, thank you!

**Reply by Lazerjaydle** [reply]
@wolveix I have tried all that I could find to set the containers timezone with no luck. I am using Unraid to run the docker container and have updated the container to the latest version. I even tried pulling from the repository you mentioned above with no luck.

**Reply by russ-sentient** [reply]
If you pull from the repo he replied to me with and add a new variable in the unraid docker configuration for that container with the key "TZ" and set the value to the timezone ("America/Chicago" for me) it should work (worked for me).

Make sure your unraid system time is set correctly also.
